src/Samples/SingleCounter/Program.fs

```diff
@@ -18,6 +18,8 @@ type Msg =
   | SetStepSize of int
   | Reset
 
+let canReset = (<>) (init ())
```

Comment:
I don't like point-free style for this free-standing function (I mostly use point-free style for inline functions). Could you write this as `let canReset m = m <> init ()` or keep inline?

Reply:
Since the plan is to change `init` from a function to a value, I definitely prefer the resulting inline expression.  I reverted this change.

Also rebased onto newest `master`.